checker/jdk/nullness/src/java/lang/Class.java

```diff
@@ -54,7 +59,7 @@ protected Class() {}
   public boolean desiredAssertionStatus() { throw new RuntimeException("skeleton method"); }
   @Pure public boolean isEnum() { throw new RuntimeException("skeleton method"); }
   public @NonNull T @Nullable [] getEnumConstants() { throw new RuntimeException("skeleton method"); }
-  java.util.Map<String, T> enumConstantDirectory() { throw new RuntimeException("skeleton method"); }
+  java.util.Map<String, @NonNull T> enumConstantDirectory() { throw new RuntimeException("skeleton method"); }
```

Comment:
I don't think every annotation in the JDK merits a test case.  An enum constant cannot be null, so this annotation is correct and necessary.